pycram/demos/pycram_multiverse_demo/demo.py

```diff
@@ -25,15 +18,15 @@
 
 milk = Object(
     "milk",
-    pycrap.Milk,
+    semantic_digital_twin.semantic_annotations.semantic_annotations.MilkContainer,
```

Comment:
Don't do imports like this, use "from semantic_digital_twin.semantic_annotations.semantic_annotations import MilkContainer"